.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Feature Request
+about: Suggest an idea for the roadmap! The team will review
+title: "[request]: describe request here"
+labels: Proposed
+assignees: ''
+
+---
+
+<!-- Please keep this note for the community -->
+
+### Community Note
```

Comment:
Can we put this at the bottom of the issue template?

Reply:
Done and done